src/views/CardStack/CardStack.js

```diff
@@ -294,7 +298,7 @@ class CardStack extends React.Component<Props> {
           ? layout.height.__getValue()
           : layout.width.__getValue();
         const currentValue =
-          I18nManager.isRTL && axis === 'dx'
+          (I18nManager.isRTL && axis === 'dx') || gestureDirectionInverted
```

Comment:
Not exactly sure what the `I18nManager.isRTL` does here. Right edfe back navigation didn't seem to work without the other fixes in this file as well. Should this be marked as a bug and handled elsewhere? Should we try to fix it here in this PR. Should we remove the attempted compatibility with RTL locales?